nxdrive/gui/application.py

```diff
@@ -667,6 +667,7 @@ def show_conflicts_resolution(self, engine: Engine) -> None:
         self.refresh_conflicts(engine.uid)
         self._window_root(self.conflicts_window).setEngine.emit(engine.uid)
         self.conflicts_window.show()
+        self.conflicts_window.raise_()
         self.conflicts_window.requestActivate()
```

Comment:
Refactor for less duplicated code:
```suggestion
        self._show_window(self.conflicts_window)
```